exercises/10/readme.md

```diff
@@ -11,6 +11,9 @@ At the end of these steps, your project will be deployed to SAP Cloud Platform C
 ### 1. Sign in to Cloud Foundry
 
 :point_right: Run the following command from your command line to log in to the referenced Cloud Foundry endpoint. When prompted use your SAP Cloud Platform credentials
+
+> If you are surprised what your SAP Cloud Platform credentials are, make sure you have a SAP Cloud Platform Cloud Foundry Trial as described in the [Prerequisites - SAP Cloud Platform](https://github.com/SAP-samples/cloud-cap-nodejs-codejam/blob/master/prerequisites.md#cloud-platform).
```

Comment:
I think "surprised" isn't the best word in this context

Reply:
Yes, I'm not sure what "surprised" means / implies here - can you please expand, @maxstreifeneder?

Reply:
People happen to log in via CF CLI to their Cloud Foundry Trial with the comment "what are my credentials??" and it turns out, that they don't have a SCP CF Trial at all. surprised was the wrong wording. Replaced it "if you are not sure what your ... credentials"